tests/integration/targets/monit/files/httpd_echo.py

```diff
@@ -0,0 +1,57 @@
+import sys
```

Comment:
I'm not at all attached to these python files (httpd_echo.py and daemon.py). I just needed a process that monit could monitor and wanted something very simple and without any other dependencies.